sql/parser/builtins.go

```diff
@@ -294,7 +294,7 @@ var builtins = map[string][]builtin{
 
 	"translate": {stringBuiltin3(func(s, from, to string) (Datum, error) {
 		const deletionRune = utf8.MaxRune + 1
-		translation := make(map[rune]rune, len(from))
+		translation := make(map[rune]rune, utf8.RuneCountInString(from))
```

Comment:
does this really make sense? After all, the cap is just a size hint in the case of maps, and I would guess that duplicate runes would not immediately deplete whatever the map internally translated the capacity to (haven't actually checked what it does, of course).